tools/tertiary-analysis/scanpy/scanpy_macros2.xml

```diff
@@ -1,10 +1,12 @@
 <macros>
-  <token name="@TOOL_VERSION@">1.8.1+3</token>
+  <token name="@TOOL_VERSION@">1.8.1+4</token>
```

Comment:
We need to move out of this pattern as it confuses the tools sorting. Please remove the +<digit> here and simply bump the Galaxy build number for the tools.

Reply:
See #266